terraform/main.tf

```diff
@@ -431,8 +431,81 @@ provider "helm" {
   }
 }
 
+# Prometheus alertmanager must be hooked up to VictorOps to deliver alerts,
```

Comment:
Could the monitoring stuff live inside another module? I'm trying to cleanup `main.tf` to basically only invoke other modules.